Clinical-Genomics/scout
# Fix #2305 - IGV zoom length contig

This PR fixes a bug.

**How to test**:
1. on a case with cram files available, verify that opening IGV view on the whole length of a SV variant (del, dup, etc) opens IGV on all configs
![Screenshot 2021-01-26 at 15 09 24](https://user-images.githubusercontent.com/758570/105855612-90b89780-5fe8-11eb-8e52-0fdb80559db3.png)

1. after applying patch, note that IGV instead opens on the correct chromosome and region
 
![Screenshot 2021-01-26 at 15 09 03](https://user-images.githubusercontent.com/758570/105855634-957d4b80-5fe8-11eb-9144-a75fb9226cd3.png)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
